docker-compose.yml

```diff
@@ -28,9 +28,9 @@ services:
       - "9229:9229"
     environment:
       NODE_PORT: 3004
-      NODE_ENV: "development"
+      NODE_ENV: "${NODE_ENV:-development}"
```

Comment:
This will read from the top-level `NODE_ENV`, and if none is present, or if the value is blank, will fall back to `development`